types/debounce-promise/index.d.ts

```diff
@@ -1,26 +1,45 @@
 // Type definitions for debounce-promise 3.1
 // Project: https://github.com/bjoerge/debounce-promise
 // Definitions by: Wu Haotian <https://github.com/whtsky>
+//                 Su <https://github.com/wowissu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
 declare namespace debounce {
-    interface DebounceOptions {
+    interface DebounceOptionsNonAccumulate {
         leading?: boolean;
         accumulate?: boolean;
```

Comment:
might be `accumulate?: false` ?

Reply:
Your right, for a clear definition, it should be.